vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -47,16 +49,66 @@ type InstallerController struct {
 
 	// installerPodImageFn returns the image name for the installer pod
 	installerPodImageFn func() string
+
+	// installStateFunc derives the static pod install state of a node
+	staticPodStateFunc StaticPodStateFunc
+}
+
+// StaticPodState is the status of a static pod that has been installed to a node.
+type StaticPodState int
+
+const (
+	// staticPodStatePending means that the installed static pod is not up yet.
+	StaticPodStatePending = StaticPodState(iota)
+	// StaticPodStateReady means that the installed static pod is ready.
+	StaticPodStateReady
+	// StaticPodStateFailed means that the static pod installation of a node has failed.
+	StaticPodStateFailed
+)
+
+const deploymentIDLabel = "deployment-id"
```

Comment:
nit: did we ended up calling this "revision" ? `deployment-revision` ?

Reply:
still seeing deployment-id in the code